src/WebJobs.Script/Description/ScriptFunctionInvokerBase.cs

```diff
@@ -79,6 +79,14 @@ protected override Task InvokeCore(object[] parameters, FunctionInvocationContex
                             Value = stream
                         };
                         await inputBinding.BindAsync(bindingContext);
+
+                        if (bindingContext.Value is string)
```

Comment:
>if (bindingContext.Value is string) [](start = 24, length = 35)

Wondering if ts this the right place for this check. No of the other type checks are here.